src/java/org/apache/cassandra/db/filter/RowFilter.java

```diff
@@ -98,18 +100,11 @@ public List<Expression> getExpressions()
         return root.expressions;
     }
 
-    /**
-     * @return *all* the expressions from the RowFilter tree in pre-order.
-     */
-    public Stream<Expression> getExpressionsPreOrder()
-    {
-        return root.getExpressionsPreOrder();
-    }
-
     /**
      * Checks if some of the expressions apply to clustering or regular columns.
      * @return {@code true} if some of the expressions apply to clustering or regular columns, {@code false} otherwise.
      */
+    @SuppressWarnings("BooleanMethodIsAlwaysInverted")
```

Comment:
While I see the point of these warning for enable/disable methods and things like that, I don't see any advantage in renaming this to something like `hasNotExpressionOnClusteringOrRegularColumns`. So I'll leave the warning in case someone someday figures out a better method name. We have been manually ignoring it for the last seven years, I guess we can keep that way.